joomla/joomla-cms
# [com_users] user groups: use the same visual level structure as categories

#### Summary of Changes

This PR does for user groups, the same visual change in the tree structure as done:
- for categories https://github.com/joomla/joomla-cms/pull/10213
- for tags https://github.com/joomla/joomla-cms/pull/10526

After PR
![image](https://cloud.githubusercontent.com/assets/9630530/15323327/608651b2-1c39-11e6-9d4d-93ffd1aa13fa.png)
#### Testing Instructions
1. Use latest staging
2. Apply patch
3. Go to Users -> Groups and check the tree structure visual
4. Go to Users -> Manage, create/edit a user, and in "Assigned User Groups" check the tree structure visual
5. Go to Users ->Access Levels, create/edit a access level, and in "User Groups Having Viewing Access" check the tree structure visual.

Note: in this PR, the HTML the tree structure prefix was moved to a layout file so it can be easily overrided or changed across all tree structures.
After (and if) this PR is merged will update tags and categories to also use the layout.
